doc/pages/JenkinFeatures.md

```diff
@@ -0,0 +1,145 @@
+# Jenkins Features Reference
+The "Jenkins Features Reference" is a comprehensive documentation outlining the various [configuration parameters](https://ci.adoptium.net/view/Test_grinder/job/Grinder/build?delay=0sec) and options available for running [Grinder jobs](https://ci.adoptium.net/view/Test_grinder/job/Grinder/) on the Jenkins server.
+It serves as a guide for testers and developers using AQAvit Grinder to perform testing and triage on the ci.adoptium.net Jenkins server and the OpenJ9 Jenkins server (and internal servers).
+The reference provides detailed explanations and instructions for each configuration parameter, enabling users to customize their test jobs effectively.
+
+## Access Permissions
+For access permissions to the Grinder job at ci.adoptium.net, you need to be part of the `test-triage` Github team (requires [2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) on your Github account).
+
+## Grouping and Granularity
+We have many tests, so it is important to be able to slice and dice them in different ways to be efficient with testing, debugging, and triage.
+Tests are organized into groups typically based on where they came from and what type of test they are: `openjdk`, `perf`, `system`,  `functional`, `external`.
```

Comment:
We can remove this section as its answering "why" and is covered under other documentation versus being a reference on implementation details of how we run tests in a Jenkins environment.